File: codemap/indexer/testdata/scala/src/main/scala/acme/store/Repo.scala
package acme.store

import acme.orders.Order

trait Repo {
  def save(order: Order): Int
}

class PgRepo extends Repo {
  override def save(order: Order): Int = insert(order)
  def insert(order: Order): Int = order.id
}

object PgRepo {
  val table = "orders"
}

File: codemap/indexer/testdata/kt/app/src/main/kotlin/acme/store/Repo.kt
package acme.store

import acme.orders.Order

interface Repo {
    fun save(o: Order): Int
}

class PgRepo : Repo {
    override fun save(o: Order): Int = insert(o)

    fun insert(o: Order): Int = o.id

    companion object {
        fun table() = "orders"
    }
}
